hardhat.config.ts

```diff
@@ -57,6 +57,11 @@ const config: HardhatUserConfig = {
       url: 'https://rpc.sepolia.org',
       accounts: [sepoliaPrivateKey],
     },
+    sepolia2: {
```

Comment:
why? they are the same (sepolia and sepolia2)

Reply:
To have deployment data for test and stage environment